kyaml/yaml/match_test.go

```diff
@@ -45,35 +45,56 @@ spec:
 	}{
 		{[]string{
 			"spec", "template", "spec", "containers", "[name=.*]"},
-			"- name: nginx\n  image: nginx:1.7.9\n  ports:\n  - containerPort: 80\n" +
-				"- name: sidecar\n  image: sidecar:1.0.0\n  ports:\n  - containerPort: 8081\n  - containerPort: 9090\n"},
+			`- name: nginx
```

Comment:
TBH I'd just leave these alone for now.  This PR is easier to digest when there are not formatting changes introduced that are made for human readability.

Reply:
done. There is no diff for this file now, thanks!